### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by shubhrsingh22**

### CODE REVIEW

**Review by danstowell — COMMENTED** [review]
LGTM!

File: baselines/deep_learning/util.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    return loss_val, acc_val
+
+
+def get_probability(x_pos,neg_proto,query_set_out):
+
+
+    """Calculates the  probability of each query point belonging to either the positive or negative class
+     Args:
+     - x_pos : Model output for the positive class
+     - neg_proto : Negative class prototype calculated from randomly chosed 100 segments across the audio file
```
Comment (veronicamorfi) [inline baselines/deep_learning/util.py:85]: Is this still from 100 segments? It seems like it might be too many?

Comment (veronicamorfi) [reply baselines/deep_learning/util.py:85]: Would it be too much to have multiple negative prototypes. that each is compute from 5 samples. and then compute the distance from each of them and the distance from the positive and if get the probabilities etc like normal?

**Review by veronicamorfi — COMMENTED** [review]
Added on comment on util.py

**Pull request merged**